index.html

```diff
@@ -12,40 +12,38 @@
     
     <!-- Code will go here for the bootstrap template-->
 
-<!--Navbar-->
-    <nav id="top-navbar" class="navbar navbar-custom">
+    <nav class="navbar navbar-expand-lg navbar-light bg-custom">
         <div class="container-fluid">
-            <!-- Logo-->
-            <div class="navbar-header">
-                <section>
-                    <img src="images/robologo.png" class="roborox-logo" alt="Roborox logo">
-                    <a class="navbar-brand" href="#">RoboRox</a>
-                </section>
-            </div>
-            <ul class="nav">
-                <li>
-                    <!-- Search bar at the top starts here --> 
-                    <!-- <div id="search-bar-top" class="search-bar-top"> -->
-                        <div id="header-search" class="header-search d-none"> <!--d-none went on this line-->
-                                <input id="search-header" type="text" placeholder="Search">                
-                            <button id="top-button-search" type="button" class="btn-primary search-image"> </button>
-                        </div> <!-- Top of search bar end-->
-                    <!-- </div> -->
-                </li>
-                <li class="nav-item">
-                <a class="nav-link" href="index.html" target="_blank">Home</a>
-                </li>
-                <li class="nav-item">
-                <a class="nav-link" href="index.html" target="_blank">About Us</a>
-                </li>
-                <li class="nav-item">
-                <a class="nav-link" href="index.html" target="_blank">Chris' Blog</a>
-                </li>
-                </li>
+          <a class="navbar-brand" href="index.html">Roborox</a>
+          <!-- Hamburger button-->
+          <button id="hamburger-button"class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent">
+              <!-- The navbar supposed content part here is referencing the ID of the stuff in the navbar below so that when it goes to hamburger mode, it knows to put the entire div inside of it-->
+            <span class="navbar-toggler-icon"></span>
+          </button>
+          <!-- End hamburger button-->
+
+          <!-- Start search bar-->
+          <form id="header-search" class="d-flex d-none">
+            <input class="form-control" type="search" placeholder="Search">
```

Comment:
@engle978 this  type="search"  is new to me, but seems like a good idea.  Just try to be consistent and apply same  type="search" to the other search textbox on the page.